features/claim-mgt/org.wso2.carbon.claim.mgt.server.feature/resources/conf/claim-config.xml

```diff
@@ -1599,12 +1605,12 @@
 			</Claim>
 			<Claim>
 				<ClaimURI>website</ClaimURI>
-				<DisplayName>URL</DisplayName>
-				<AttributeID>url</AttributeID>
+				<DisplayName>Website URL</DisplayName>
+				<AttributeID>websiteUrl</AttributeID>
```

Comment:
As this attributeID is not present in embedded LDAP schema we use, default pack can get issues. Let's create an issue to add this attribute to default schema as well.

Reply:
https://github.com/wso2-extensions/identity-userstore-ldap/pull/46 done here